tsconfig.json

```diff
@@ -14,10 +14,9 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es2015",
+    "target": "es5",
```

Comment:
is it necessary to lower target version?

Reply:
Yes